packages/endpoints-overlay/src/endpoint.tsx

```diff
@@ -176,7 +177,10 @@ const Endpoint = (props: Props): JSX.Element => {
           latitude={location.lat}
           longitude={location.lon}
         >
-          <div>
+          <FocusTrapWrapper
```

Comment:
Not seeing the popup for `Endpoint`, also not on the master branch. Should we fix that?

Reply:
Do you mean the popup isn't showing up in the story?

Reply:
Correct

Reply:
Fixed in [2e969ce](https://github.com/opentripplanner/otp-ui/pull/743/commits/2e969ce70f0c33f05265da21da83f7dde0953917)